test/eviction/evictpod.go

```diff
@@ -49,8 +51,9 @@ func main() {
 	if *pod == "" {
 		log.Fatal("--pod must be non-empty")
 	}
+	logger := logrus.New()
```

Comment:
```suggestion
   logger = runtime.NewLoggerWithSource("main")
```

To be consistent. Might even switch out the `log.` below to `logger` to be nice and clean.

Reply:
All the changes are in place per your suggestion👍